.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
Please remove this, you should add this in a global gitignore

https://murze.be/2014/12/create-a-global-gitignore/